docs/os-ref.md

```diff
@@ -6,28 +6,28 @@ sidebar_label: Reference documentation
 
 *Documentation for [aragonOS](https://github.com/aragon/aragonOS) v4.0.1. Looking for [aragonOS 3 documentation?](/docs/aragonos-3-ref.html)*
 
-This document provides a technical overview of the framework's architecture and provides insight into its capabilities. It assumes the reader to have an understanding of [Solidity](https://solidity.readthedocs.io/). For a less technical introduction, visit the [introduction](/docs/aragonos-intro.html).
+This document provides a technical overview of the framework's architecture and provides insight into its capabilities. It assumes the reader understands [Solidity](https://solidity.readthedocs.io/). For a less technical introduction, visit the [introduction](/docs/aragonos-intro.html).
 
 ## Design philosophy
 
 Using aragonOS allows you to write simpler code by **decoupling** the specific **business logic** of a protocol or application from its **authentication logic**.
 
-You shouldn't have to think about how to technically implement authentication or governance at all; simply inherit from the **AragonApp** base class and define actions that require authentication with a special modifier. The rest is taken care of by the components provided in aragonOS.
+With aragonOS, you don't have to think about how to implement authentication or governance at all. Simply inherit from the **AragonApp** base class and use a special modifier to mark actions that require authentication. 
 
-Additionally, **upgradeability** capabilities are provided and are used by default. aragonOS implements the [DelegateProxy](https://eips.ethereum.org/EIPS/eip-897) pattern, and specifically a special implementation called [unstructured storage](https://blog.zeppelinos.org/upgradeability-using-unstructured-storage/). This pattern essentially splits an contract into two instances: a **base logic contract**, which is then depended upon by a simple, slim **proxy contract**. The proxy delegates all its logic to the linked base contract, and can modify its pointer to the base contract, effectively upgrading its logic.
+Additionally, **upgradeability** capabilities are provided and are used by default. aragonOS implements the [DelegateProxy](https://eips.ethereum.org/EIPS/eip-897) pattern known as  [unstructured storage](https://blog.zeppelinos.org/upgradeability-using-unstructured-storage/). This pattern essentially splits a contract into two instances: a **base logic contract**, which is then depended upon by a simple, slim **proxy contract**. The proxy delegates all its logic to the linked base contract and can modify its pointer to the base contract in order to upgrade its logic.
```

Comment:
@cjyabraham I wouldn't necessarily group unstructured storage with the `DelegateProxy`, as unstructured storage is technically only one possible implementation of the `DelegateProxy` pattern.

Reply:
Ok, I've changed to this:
"aragonOS implements the DelegateProxy pattern with a special implementation called unstructured storage."

fixed in 4ad2ab5